src/components/custom-sdk/template/HMRC_ODX_GDSTaskListTemplate/index.tsx

```diff
@@ -39,6 +39,10 @@ export default function HmrcOdxGdsTaskListTemplate(props: HmrcOdxGdsTaskListTemp
     }
   });
 
+  useEffect(() => {
+    sessionStorage.removeItem('assignmentID');
```

Comment:
Will this throw any undefined error , if we land on task list page for very first time .. ?